pyliquibase/__init__.py

```diff
@@ -36,7 +36,6 @@ def __init__(self, defaultsFile: str,
                  liquibaseHubMode: str = "off",
                  logLevel: str = None,
                  liquibaseDir: str = None,
-                 jdbcDriversDir: str = None,
```

Comment:
Could we leave this for end user in-case user want to provide jdbcDriversDir dir?